immuno_graphical/verification/verification_plan.Rmd

```diff
@@ -1,7 +1,9 @@
 ---
-title: Verification Plan for Immunogenicity Plots
-date: "`r format(Sys.time(), '%B %d, %Y')`"
-output: pdf_document
+title: "Verification Plan for Immunogenicity Plots"
+output:
+  html_document: default
```

Comment:
hey @dilu963 - could you remove html_document as an output? and the html document that you produced?